OSpackages.sh

```diff
@@ -318,8 +331,8 @@ function FetchCustomRepos {
 ## Main program-flow
 ######################
 OPTIONBUFR=$( getopt \
-   -o a:Fg:hm:r:M: \
-   --long help,mountpoint:,repo-activation:,repo-rpms:,rpm-group: \
+   -o a:eg:hm:r:M: \
```

Comment:
Huh. I wonder what `F` was supposed to be? @ferricoxide?

Reply:
I'm assuming copy-paystah error:

~~~
$ grep -- -F *
DiskSetup.sh:                  MKFSFORCEOPT="-F"
PostBuild.sh:      printf '\t%-4s%s\n' '-F' 'Disable FIPS support (NOT IMPLEMENTED)'
PostBuild.sh:      printf '\t%-20s%s\n' '--no-fips' 'See "-F" short-option'
PostBuild.sh:      -F|--no-fips)
~~~